test/extended/main.go

```diff
@@ -10,3 +10,21 @@ var _ = g.Describe("[Jira:kube-controller-manager][sig-api-machinery] sanity tes
 		o.Expect(true).To(o.BeTrue())
 	})
 })
+
+var _ = g.Describe("[Jira:kube-controller-manager][sig-api-machinery] serial test", func() {
+	g.It("should run serially [Serial][Suite:openshift/cluster-kube-controller-manager-operator/conformance/serial]", func() {
+		o.Expect(true).To(o.BeTrue())
+	})
+})
+
+var _ = g.Describe("[Jira:kube-controller-manager][sig-api-machinery] slow test", func() {
+	g.It("should be marked as slow [Slow][Suite:openshift/cluster-kube-controller-manager-operator/optional/slow]", func() {
+		o.Expect(true).To(o.BeTrue())
+	})
+})
+
+var _ = g.Describe("[Jira:kube-controller-manager][sig-api-machinery] disruptive test", func() {
+	g.It("should be marked as disruptive [Serial][Disruptive][Suite:openshift/cluster-kube-controller-manager-operator/conformance/serial]", func() {
+		o.Expect(true).To(o.BeTrue())
+	})
+})
```

Comment:
We only need one test to confirm the usability of test extension architecture, which is sufficient. No need more.

Reply:
updated